app/cmd/rest.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func StartRest() {
+	controller()
+}
+
+func controller() {
+	http.HandleFunc("/health", health)
+	http.HandleFunc("/hello-world", helloWorld)
+
+	fmt.Print("Starting server on port 3000\n")
+	err := http.ListenAndServe(":3000", nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func health(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" && r.URL.Path != "health" {
+		errorHandler(w, r, http.StatusNotFound)
+		return
+	}
```

Comment:
- can use `http.MethodGet` constant
- http package also had `http.NotFound` handler to use
- no need to check the path again, it already check on `HandleFunc`

Reply:
niceee thank you sire